vllm/model_executor/model_loader/loader.py

```diff
@@ -91,14 +91,15 @@ def _get_quantization_config(
     """Get the quantization config."""
     if model_config.quantization is not None:
         quant_config = get_quant_config(model_config, load_config)
-        capability = current_platform.get_device_capability()
-        capability = capability[0] * 10 + capability[1]
-        if capability < quant_config.get_min_capability():
-            raise ValueError(
-                f"The quantization method {model_config.quantization} is not "
-                "supported for the current GPU. "
-                f"Minimum capability: {quant_config.get_min_capability()}. "
-                f"Current capability: {capability}.")
+        if not current_platform.is_tpu():
```

Comment:
As a follow up (not in this PR) - we should modify `quant_config.get_min_capability()` to instead expose `quant_config.is_supported_on_platform()` such that we can avoid the conditional branching here

I think this will become important as we expand into more methods

Also - is there a min TPU version needed here?

Reply:
Agreed, this is a good point! Right now most of the version checks are specific to GPU, which is not that extensible for non-GPU hardware.

> is there a min TPU version needed here?

No, we don't need a min TPU version here.